### ISSUE TITLE
Windows line breaks break tile-extruder in Mac environment

### ISSUE THREAD

**Issue by carystanley** [sporadic-labs/tile-extruder#10]
First, thank you so much for making this tool, it's a huge benefit to everyone!!!

The file /bin/tile-extruder uses window like breaks, which causes the following error under "Mac", when I execute tile-extruder inside the scripts in my `package.json`:

`env: node\r: No such file or directory`

**Workaround for those on mac:**

`vi ./node_modules/tile-extruder/bin/tile-extruder`
then you can fix the line endings of the file with
`:set ff=unix`
then save and quit
`:wq`
then everything works again

I would have sent a PR to fix it, but when I clone the repo, my settings correct it locally, so I am unable to make the PR myself.

`md5 ./node_modules/tile-extruder/bin/tile-extruder`
MD5 (./node_modules/tile-extruder/bin/tile-extruder) = 1b0f4d12d58fbe6994d38af28b12e750

`md5 tile-extruder` (with corrected line breaks)
MD5 (tile-extruder) = 565bd371fccb8ed11b5cc14b5a6449ef

Thanks Again!!!

**Comment by mikewesthad** [comment]
Glad you found it helpful!

Hmm, so you install it locally with npm and then add an npm script like `"extrude": "tile-extruder -w 16 -h 16 ..."` and it breaks when you run `npm run extrude`? I'm trying to replicate on my linux box, but it is working via an npx command, a local npm install and an npm script.

**Comment by mikewesthad** [comment]
Thanks for reporting this. I think I've got it working in the latest release (2.0.6) on my linux machine, but if you have a minute and could test it, that would be great.

I wasn't able to replicate this issue with npm + linux, but I was able to see the issue under yarn + linux. To fix it, I added a .gitattributes to force LF endings on bin/tile-extruder, which should hopefully keep that file as LF even if it is edited in the future on a Windows machine.
